### PR TITLE
Add Zone.js detection

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

**Comment by jvoisin** [comment]
It would be nice if you could split the angular thingy into an other PR.

File: src/apps.json [hunk 1]
```diff
@@ -11415,6 +11420,16 @@
       },
       "website": "http://django-blog-zinnia.com"
     },
+    "Zone.js": {
+      "cats": [
+        12
+      ],
+      "js": {
+        "Zone": "",
```
Comment (jvoisin) [inline src/apps.json]: If `Zone.root` is always available, there is no need to have `Zone` as well.

**Comment by arturovt** [comment]
@jvoisin Okay, I will remove `Zone` and split

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**